README.md

```diff
@@ -271,6 +281,7 @@ Inspired by [awesome-python](https://github.com/vinta/awesome-python).
 * [Hermione](https://github.com/a3data/hermione) - Help Data Scientists on setting up more organized codes, in a quicker and simpler way.
 * [Hydra](https://github.com/facebookresearch/hydra) - A framework for elegantly configuring complex applications.
 * [Koalas](https://github.com/databricks/koalas) - Pandas API on Apache Spark. Makes data scientists more productive when interacting with big data.
+* [Losswise](https://losswise.com) - Makes it easy to track the progress of a machine learning project.
```

Comment:
From the description below (extracted from the Losswise website) it seems it has more fit in the "Model Lifecycle" section, what do you think?

> Losswise provides ML practioners with an intuitive and elegant Python API and accompanying dashboard to visualize progress within training sessions as well as across training sessions.
It's ideal for running hyperparameter searches, model comparison experiments, as well as simply tracking one-off training scripts.